Sources/edge/Commands/Devices.swift

```diff
@@ -0,0 +1,171 @@
+import ArgumentParser
+import IOKit
+import IOKit.usb
+import Network
+import Foundation
+import SystemConfiguration
+import Logging
```

Comment:
This will only work on macOS, can we gate it behind `#if os(macOS)`?

Reply:
yea, I probably need some protocol and and use a specific implementation based on os.